lib/active_model_serializers/adapter/json_api.rb

```diff
@@ -294,7 +294,7 @@ def attributes_for(serializer, fields)
 
       # {http://jsonapi.org/format/#document-resource-objects Document Resource Objects}
       def resource_object_for(serializer)
-        resource_object = serializer.cache_check(self) do
+        resource_object = serializer.fetch(self) do
```

Comment:
Just interjecting that you definitely want to add a `fetch_multi`, without it every fetch is doing a round trip. Also, if all of the benchmarks are using the `MemoryStore` you won't get very realistic results because wire time isn't included.